### ISSUE TITLE
Multi Org Phase 2 - Enable duplicate org names in the same region across different accounts

### ISSUE THREAD

**Issue by juliajames12** [influxdata/ui#4951]
ON HOLD - The system doesn't support this yet. 

Waiting on: https://github.com/influxdata/idpe/issues/14999  (Adam, Stacie, & Vinay))

**Comment by ALuckette** [comment]
Enabling duplicate org names per account is dependent on the compute team:

https://github.com/influxdata/idpe/issues/15596
https://github.com/influxdata/idpe/issues/15576
https://github.com/influxdata/idpe/issues/15597
https://github.com/influxdata/idpe/issues/15598

FYI @kristinarobinson @hoorayimhelping @ChitlangeSahas @wdoconnell

**Comment by wdoconnell** [comment]
For feature context - I think this means we can't implement the 'duplicate of same org in current account' error message yet (which relies on assumption that the orgs retrieved for this account from quartz represent the only constraints on org names).

**Comment by ALuckette** [comment]
Yes, that's correct. Thank you for typing that in here! :)

**Comment by wdoconnell** [comment]
This was a placeholder for idpe work, which was completed by @saraghds
